core/forms/schemas/auth-schema.ts

```diff
@@ -1,8 +1,45 @@
-import * as yup from 'yup';
```

Comment:
I dont think we need a folder for this.

Can be call `auth.schema` for example in the parent folder.

I will call the forms `login.form` isntead of `login-form` to keep the style with the `module` and other naming